src/Microsoft.VisualStudio.Editors/DesignerFramework/AccessModifierComboBox.vb

```diff
@@ -438,7 +441,23 @@ Namespace Microsoft.VisualStudio.Editors.DesignerFramework
 
             For Each codeGenerator As CodeGenerator In _codeGeneratorEntries
                 If codeGenerator.DisplayName.Equals(value, StringComparison.CurrentCultureIgnoreCase) Then
-                    TrySetCustomToolValue(codeGenerator.CustomToolValue)
+                    Dim Designer = CType(RootDesigner, ResourceEditorRootDesigner)
+                    If Designer IsNot Nothing Then
+                        Dim ResourceView As ResourceEditorView = Designer.GetView()
+                        ' We let the base class handle the read only mode
+                        ' As mentioned in ResourceEditorDesignerLoader, "We actually don't want users to edit Form RESX file"
+                        ' so we just need to add the same warning as there for now until the new resource explorer is released
+                        If ResourceView IsNot Nothing AndAlso Not ResourceView.ReadOnlyMode
+                           If ResourceView.DsMsgBox(My.Resources.Microsoft_VisualStudio_Editors_Designer.RSE_Err_UpdateADependentFile, MessageBoxButtons.YesNo, MessageBoxIcon.Warning, MessageBoxDefaultButton.Button2, HelpIDs.Err_EditFormResx) = DialogResult.Yes Then
```

Comment:
I wonder what the significance of `HelpIDs.Err_EditFormResx` is. Does that link to some documentation that'd need to be updated?

Reply:
It does not - this addresses a gap when editing a resx that shouldn't be edited - the warning that should be displayed was not. So no documentation needs to be updated